lib/iris/fileformats/netcdf/loader.py

```diff
@@ -247,17 +260,22 @@ def _get_cf_var_data(cf_var, filename):
             dim_chunks = CHUNK_CONTROL.var_dim_chunksizes.get(
                 cf_var.cf_name
             ) or CHUNK_CONTROL.var_dim_chunksizes.get("*")
-            if not dim_chunks:
+            dims = cf_var.cf_data.dimensions
+            if CHUNK_CONTROL.mode == CHUNK_CONTROL.Modes.FROM_FILE:
```

Comment:
```suggestion
            if CHUNK_CONTROL.mode == ChunkControl.Modes.FROM_FILE:
```

See my comment about a class variable instead of an instance variable (and sorry for getting this wrong when I was demonstrating!).